### ISSUE TITLE
Accessibility-Large Text not visible

### ISSUE THREAD

**Issue by timo54** [rokwire/illinois-app#17]
### Android 2.2.11 

**Describe the bug**
When font sizes are increased on the Android screen, some screens do not reflow the text so that the large font is visible. It is not possible to move the screen to the right to see this text. This is a Priority 1 issue, it must be fixed. 

**To Reproduce**
1. In Accessibility Settings, set font size to largest available
2. Go to Dining and pick a restaurant
3. Scroll through the screen to see times as shown in the screen capture below
4. Times and text do not reflow so they are visible

**Expected behavior**
Screen text and buttons must reflow to adapt to large screen fonts.

**Screenshots**
See below.

**Desktop (please complete the following information):**
 - OS: Android
 - Version 2.2.11

**Smartphone (please complete the following information):**
 - Device: Google Pixel 3
 - OS: Android
 - Version 2.2.11

<img width="433" alt="horizontal-lrgTxt-mrkup" src="https://user-images.githubusercontent.com/12497650/91499311-d8df5c80-e886-11ea-97c7-0f78c1fb0a8b.png">

**Comment by TodorBachvarov** [comment]
This was fixed. The text show ellipsis, and the time intervals are properly shown when dropdown menu is clicked.

**Comment by acovher2** [comment]
- Still present in version 2.5.9

**Comment by acovher2** [comment]
Verified as resolved in 2.6.28
